scripts/data/regions.json

```diff
@@ -561,10 +612,12 @@
     "country_codes": [
       "AE",
       "BH",
+      "CY",
       "DJ",
       "DZ",
       "EG",
       "EH",
+      "IL",
```

Comment:
Israel ? Not quite sure about this one.

Reply:
https://www.iban.com/country-codes

Reply:
Yes, that confirms what I was saying. IL is Israel, it is not a country around the LATAM region, but on the Middle East one.

Reply:
Please check again. These changes have nothing to do with the LATAM region.